docs/start/myths/overlays.md

```diff
@@ -5,12 +5,61 @@ parent: Debunking myths
 nav_order: 3
 ---
 
-# Overlays
+# Accessibility Overlays
+
+An accessibility overlay is scripted code that is placed on a web page that automatically makes changes to the page with the intent of fixing accessibility issues on that page.
+
+This purely technical definition of an overlay doesn't impose any judgement on the effectiveness of the concept: an overlay can legitimately fix accessibility problems.
```

Comment:
About
> This purely technical definition of an overlay doesn't impose any judgement on the effectiveness of the concept: an overlay can legitimately fix accessibility problems.

Although the content is absolutely right, as a not native English speaker it takes me a few times to read to understand the meaning. The English is very high level.

Is it possible to target a more international audience and scale the level of the language a bit down to B2 or so?

Reply:
The rest is perfect.

Reply:
Rephrased to "That's a technical definition of an overlay, which doesn't address the effectiveness of the overlay as a concept. An overlay can fix accessibility problems.', which is hopefully clearer.